src/include/pg_config_manual.h

```diff
@@ -57,7 +57,7 @@
  * version.  Example: "ACME Postgres/1.2".  Note that the string will appear
  * in a user-facing error message if an ABI mismatch is detected.
  */
-#define FMGR_ABI_EXTRA		"Neon Postgres"
+#define FMGR_ABI_EXTRA		"Databricks Postgres"
```

Comment:
We can't change this without breaking intercompatibility with extensions that were compiled with the other string.

@tristan957 do you know if we recompile all dynamically loaded extensions, or if we reuse those files?

If we reuse them, then this is a no-go.

Reply:
We do recompile all remote extensions for each release.

Reply:
To reduce the diff on the naming front, we can submit PRs over to the Databricks fork of Postgres.

Reply:
Changed back, I don't think there is a strong enough argument to keep that change.